paper.bib

```diff
@@ -1,10 +1,9 @@
 
-# About
-Bibliography relevant to the paper on Visualization for Python and Flask-based Web Services
+% About Bibliography relevant to the paper on Visualization for Python and Flask-based Web Services
```

Comment:
Mr. @FlyingBird95 : it would be ok to keep the markdown type 
of annotations.  bib happily ignores them.
then all we need to do is to provide a symbolic link to the bib
that is named bibliography.md and it can be nicely rendered
as rich text by md compiler ;) 

see this: https://github.com/mircealungu/mir-biblio/blob/master/aslan.bib.md